File: territoire-front/models/person.ts
export interface Person {
    id: string | null;
    firstName: string;
    lastName: string;
    email?: string;
    phoneNumber?: string;

}

export function personToString(person?: Person):string {
    if (!person) {
        return "N/A";
    }
    return `${person.firstName} ${person.lastName}`;
}

File: territoire-back/src/main/java/com/teillet/territoire/model/Person.java
package com.teillet.territoire.model;

import jakarta.persistence.*;
import lombok.*;

import java.util.UUID;

@Entity
@Getter
@Setter
@ToString
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Person {
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	private UUID id;

	private String firstName;
	private String lastName;
	private String phoneNumber;
	private String email;
}
